### PR TITLE
Update README

### LINKED ISSUE #62: Advertise installation via AUR
Arch Linux users  can also use this AUR to install reuse: https://aur.archlinux.org/packages/reuse

We should mention it in the README, the docs, and the website.

### PR DESCRIPTION
**Pull request by mxmehl**
This updates the README with new content on the website. It also improves smaller formatting/wording things.

Fixes #62

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,17 +38,17 @@ As a short summary, the recommendations are threefold:
 3.  Confirm REUSE compliance
 
 You are recommended to read the
-[recommendations](https://reuse.software/practices/) in full for more
+[our tutorial](https://reuse.software/tutorial) in full for more
 details.
```
Comment (carmenbianca) [inline README.md]: "for more details" is probably inaccurate here. I'd rephrase to:

> You are recommended to read our tutorial for a step-by-step guide through these three steps. For more details, you may read the specification.

With appropriate links etc etc etc

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ SPDX-FileCopyrightText: 2017-2019 Free Software Foundation Europe e.V.
 SPDX-License-Identifier: CC-BY-SA-4.0
 -->
 
-# reuse
+# REUSE helper tool
```
Comment (carmenbianca) [inline README.md]: This tool has an identity crisis :p It is reminiscent of the EFF's "certbot" tool, which has finally settled on a name, but has also gone by the names "certbot-auto" and "letsencrypt".

Before accepting this change, I'd rather work out how to properly solve the identity crisis in a separate issue.

Comment (mxmehl) [reply README.md]: Good point. However, I think an issue won't be the best place to figure this out. I'd rather like a call with Gabriel etc where we discuss the naming of REUSE and its projects, how the team calls itself and so on. What do you think?

File: README.md (hunk 1 above)
Comment (carmenbianca) [reply README.md]: See #91

File: README.md (hunk 2 above)
Comment (carmenbianca) [reply README.md]: Ah, already created #91, but we can discuss this outside of an issue, and simply close the issue when it is solved externally ^^

File: README.md (hunk 1 above)
Comment (mxmehl) [reply README.md]: Thanks! I borrowed text from the REUSE 3.0 news item to link to tutorial, faq and spec

**Comment by mxmehl** [comment]
I also worked on the installation part, which could close #62. I'm not sure about the exact wording for PIP and via source code, so please review

**Comment by carmenbianca** [comment]
Fixes #62 

Didn't really need to change the wording. I'm currently in the process of updating the tool for Fedora:

https://bugzilla.redhat.com/show_bug.cgi?id=1748059

https://bugzilla.redhat.com/show_bug.cgi?id=1748059

^ Dependencies before I can update the tool.

I'll also have a look at openSUSE again.
